uber/models.py

```diff
@@ -1338,6 +1342,15 @@ def requested(self, department):
     def assigned_to(self, department):
         return int(department or 0) in self.assigned_depts_ints
 
+    def trusted_in(self, department):
+        return int(department or 0) in self.trusted_in_ints
+
```

Comment:
There are absolutely people trusted by Reg who are likely not assigned to Reg. Good folks get stolen from us all the time. :)